src/tribler/core/components/database/db/layers/health_data_access_level.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+import logging
+
+from pony import orm
+
+from tribler.core.components.database.db.layers.knowledge_data_access_layer import KnowledgeDataAccessLayer
+from tribler.core.components.torrent_checker.torrent_checker.dataclasses import HealthInfo
+from tribler.core.upgrade.tags_to_knowledge.previous_dbs.knowledge_db import ResourceType
+from tribler.core.utilities.pony_utils import get_or_create
+
+
+# pylint: disable=redefined-outer-name
+
+
+class HealthDataAccessLayer:
+    def __init__(self, knowledge_layer: KnowledgeDataAccessLayer):
+        self.logger = logging.getLogger(self.__class__.__name__)
+        self.instance = knowledge_layer.instance
+        self.Resource = knowledge_layer.Resource
+        self.HealthInfo = self.define_binding(self.instance)
+
+    @staticmethod
+    def define_binding(db):
+        class HealthInfo(db.Entity):
```

Comment:
I think the `Tracker` entity should also be added to the database. The `HealthInfo` entity should also have a `tracker` field that refers to the `Tracker` entity.

Reply:
Done